codejam-canvas/assets/css/style.scss

```diff
@@ -0,0 +1,137 @@
+$grayColor: #eeeeee;
+
+
+* {
+    box-sizing: border-box;
+  }
+  
+body{
+    max-width: 1440px;
+    margin: 0 auto;
+    background-color: #eeeeee;
+}
+
+.wrapper{
+    background-color: #ffffff;
+    padding-bottom:300px;
+}
+
+header{
+    box-shadow: 0px 3px 3px $grayColor;
+}
+
+nav{
+    padding: 15px;
+    display: flex;
+    justify-content: space-between;
+
+}
+
+main{
+    margin: 50px 20px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.left_side{
+    display:flex;
+    //justify-content: space-between;
+    width:20%;
+
+    h1{
+        padding-left: 20px;
+        font-family: Roboto;
+        color: rgba($color: #000000, $alpha: 0.87);
+        font-size: 20px;
+    }
+
+    img{
+        cursor: pointer;
+    }
+}
+
+.right_side img{
+    padding:0 10px;
+
+    cursor: pointer;
+}
+
+.left_panels_area{
+    display:flex;
```

Comment:
`display: flex;`
Don't forget to leave empty spaces between property and value.
Check it everywhere